lib/pyborg/pyborg-irc.py

```diff
@@ -375,10 +377,15 @@ def on_msg(self, c, e):
             if body[0] == "!":
                 if self.irc_commands(body, source, target, c, e) == 1:return
 
+        # Calculate delay
+        delay = self.settings.delay[0]
+        if len(self.settings.delay) == 2:
+            delay = random.randint(self.settings.delay[0], self.settings.delay[1])
```

Comment:
For even more realism I'd suggest using
```python
delay = random.uniform(self.settings.delay[0], self.settings.delay[1])
```
so that a random float is picked instead of just an integer amount of seconds.